RTC beta (#1778)
tinymce/tinymce-docs: tinydrive/integrations/googledrive-integration.md

### 6. Configure tinydrive to use the new Google Drive credentials

> You configure the Google drive API key by setting the [tinydrive_google_drive_key]({{site.baseurl}}/tinydrive/configuration/#tinydrive_google_drive_key) and the Google OAuth client ID by setting the [tinydrive_google_drive_client_id]({{site.baseurl}}/tinydrive/configuration/#tinydrive_google_drive_client_id).

Reviewer: ```suggestion
The Google drive API key can be configured by setting the [tinydrive_google_drive_key]({{site.baseurl}}/tinydrive/configuration/#tinydrive_google_drive_key). The Google OAuth client ID can be configured by setting the [tinydrive_google_drive_client_id]({{site.baseurl}}/tinydrive/configuration/#tinydrive_google_drive_client_id).
```
maybe? feels like it needs more words still

Reviewer 2: This should also probably be `Google Drive` as it's a product name, so Drive should be captialised.

Reviewer: updated
